__tests__/bundle/bundle-lint-format/checkstyle-format-snapshot.js

```diff
@@ -9,7 +9,7 @@ exports[`E2E bundle lint format bundle lint: should be formatted by format: chec
 </checkstyle>
 
 ❌ Validation failed with 1 error.
-run \`openapi lint --generate-ignore-file\` to add all problems to the ignore file.
+run \`redocly lint --generate-ignore-file\` to add all problems to the ignore file.
```

Comment:
Could you extract those changes into separate PR plz?

Reply:
Sure. Just stored everything in the draft in order not to lose the changes.